lib/Events/SignalingEvent.php

```diff
@@ -28,7 +28,7 @@
 
 class SignalingEvent extends ParticipantEvent {
 	/** @var mixed */
-	protected $session;
+	protected mixed $session;
```

Comment:
```suggestion
	protected ?string $session;
```

But should also be enforced on the setter and getter then

Reply:
Also was not done.

Reply:
Sorry my commit wasn't push, It's fixed now